tests/_test_tpm2_file_permissions

```diff
@@ -15,7 +15,6 @@ PIDFILE=${TPM_PATH}/swtpm.pid
 LOGFILE=${TPM_PATH}/swtpm.log
 PWDFILE=${TPM_PATH}/pwdfile.txt
 SWTPM_INTERFACE=${SWTPM_INTERFACE:-cuse}
-SWTPM_SETUP_CONF=${TPM_PATH}/swtpm_setup.conf
```

Comment:
This one here will be dynamically created.

Reply:
ack